thirdparty/enet/godot.cpp

```diff
@@ -0,0 +1,196 @@
+/**
+ @file  godot.c
```

Comment:
s/godot.c/godot.cpp/ :)

Maybe add the usual Godot header to those two godot files?